src/api/apps/graphql/index.js

```diff
@@ -146,6 +146,8 @@ const schema = joiql({
   },
 })
 
+module.exports.schema = schema
```

Comment:
Weirdly this was needed to execute the script directly from node command line even though `babel-node` is being loaded before everything [here](https://github.com/artsy/positron/blob/91dcaee3c909685c7e3710302e152021c35e315d/package.json#L25)